archive/100-theorems-list/83_friendship_graphs.lean

```diff
@@ -0,0 +1,261 @@
+/-
+Copyright (c) 2020 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author:  Aaron Anderson, Jalex Stark.
+-/
+import combinatorics.adj_matrix
+import linear_algebra.char_poly.coeff
+import data.int.modeq
+import data.zmod.basic
+import tactic
+
```

Comment:
Could you write a module doc that explains what the problem is and how you've solved it?

Reply:
I've added a module doc with a proof outline.